README.md

```diff
@@ -202,7 +208,7 @@ The graph of the ROS nodes is shown below:
 One can plan a new path by setting a new goal with the *2D Nav Goal* button in rviz. The planned path should show up in rviz and the drone should follow the path, updating it when obstacles are detected. It is also possible to set a goal without using the obstacle avoidance (i.e. the drone will go straight to this goal and potentially collide with obstacles). To do so, set the position with the *2D Pose Estimate* button in rviz.
```

Comment:
What is rviz? This is the first time it is mentioned. Did we already set it up somewhere? What is it used for?

Reply:
Is it enough to reference the ROS wiki for all the ROS tools such as rviz and rqt? For example http://wiki.ros.org/rviz .

I think we have given for granted that people are somewhat familiar with ROS throughout the document.

Reply:
YOu talked about providing link to rviz and other tools. I may have missed it, but first instance should be linked. Usually subsequent instances would not be. If I was on the wiki I would suggest marking subsequent instances as *rviz* (italic) (ie generally I use italic for tools)